packages/core/src/common/_variables.scss

```diff
@@ -40,7 +40,7 @@ $pt-icon-size-large: 20px !default;
 
 // Grids & dimensions
 
-$pt-border-radius: floor($pt-grid-size / 3) !default;
+$pt-border-radius: $pt-grid-size * 0.4 !default;
```

Comment:
I think it's time to be explicit: `$pt-border-radius: 4px`. 

this math is no better than the previous iteration: if I set `$pt-grid-size: 12px` this is gonna choke up real bad (`4.8`)